src/main/java/edu/hw1/Task1.java

```diff
@@ -0,0 +1,43 @@
+package edu.hw1;
+
+import java.io.IOException;
+
+public class Task1 {
+    public final String uncorrectDigitInInput = "некоректные цифры во вводе";
```

Comment:
`BUG` в jav'e константы принято называть в **upper snake case**, т.е. **ВОТ_ТАК**, а ещё делать их static
```suggestion
    public static final String  INCORRECT_DIGIT_IN_INPUT_MESSAGE = "некоректные цифры во вводе";
```

Reply:
P.s. подобные комментарии к стилю кода и т.д. надо поправить везде